public/r2/v1/swagger/swagger.json

```diff
@@ -877,6 +943,61 @@
                 }
             }
         },
+        "RuleSetChanges": {
+            "type": "object",
+            "properties": {
+                "MappingRuleChanges": {
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/MappingRuleChange"
+                    },
+                    "description": "list of mapping rule changes"
+                },
+                "RollupRuleChanges": {
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/RollupRuleChange"
+                    },
+                    "description": "list of rollup rule changes"
+                }
+            }
+        },
+        "MappingRuleChange": {
+            "type": "object",
+            "properties": {
+                "operation": {
+                    "type": "string",
+                    "enum": ["add", "remove", "change"],
```

Comment:
Can we please change `remove` to `delete` for consistency? That's what we use in m3metrics as well.